### ISSUE TITLE
unable to unlock vault via cli

### ISSUE THREAD

**Issue by ShaunHoyes** [bitwarden/cli#78]
When I try to unlock the vault via the cli, I get the following error message:
```
Invalid master password.
```
I'm pretty certain I have the correct master password. I also have 2FA enabled. Is it a problem to use the cli with 2FA enabled?

**Comment by kspearrin** [comment]
Have you tried logging out and back in?

**Comment by ShaunHoyes** [comment]
I did try to log out and log back in, and after doing more tinkering, I just learned that it's necessary to "set your session key to the `BW_SESSION` environment variable" each time a new session is started. By doing this, I have no trouble gathering password info. 

But when I use the `bw unlock [myPassword123]` command, I still get the error message. So everything works, it's just that one command that returns an error.

Thanks!
